com.tracktion.Waveform11.yml

```diff
@@ -0,0 +1,128 @@
+id: com.tracktion.Waveform11
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform11
+finish-args:
+  - --device=all
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2
+  - --env=LXVST_PATH=/app/extensions/Plugins/lxvst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
```

Comment:
did you check that the app support LADSPA and LV2?
Also did you check it uses LXVST_PATH for VST. Sometime it's VST_PATH.

Reply:
I didn't really test it beyond noticing it loaded a bunch of plugins with these flags.

Reply:
it would be nice to know which one to remove. This is helpful down the line for maintenance.

Reply:
Cleaned them up now.